Product/pom.xml

```diff
@@ -408,26 +408,12 @@
                     </exclusion>
                 </exclusions>
             </dependency>
+		
             <dependency>
-                <groupId>org.hibernate</groupId>
-                <artifactId>hibernate-annotations</artifactId>
-                <version>3.3.1.GA</version>
-                <exclusions>
-                    <exclusion>
-                        <groupId>cglib</groupId>
-                        <artifactId>cglib</artifactId>
-                    </exclusion>
-                    <exclusion>
-                        <groupId>org.hibernate</groupId>
-                        <artifactId>ejb3-persistence</artifactId>
-                    </exclusion>
-                </exclusions>
-            </dependency>
-            <dependency>
-                <groupId>org.hibernate</groupId>
+                <groupId>org.hibernate.common</groupId>
                 <artifactId>hibernate-commons-annotations</artifactId>
-                <version>3.0.0.ga</version>
-            </dependency>
+                <version>5.0.1.Final</version>
```

Comment:
That is a supporting hibernate artifact whose version is updated so that we can get a newer release..  The main hibernate artifact (hibernate-core) is 5.1.